public-ui/src/app/views/pm/pm.component.css

```diff
@@ -0,0 +1,16 @@
+.measurement.white-card-box {
+    min-height: 400px;
+}
+.chart-wrapper {
+    width: 336px;
+}
+
+/* Style the active sub buttons */
+button.active.btn-link {
+    font-weight: bolder;
+}
+
+.row.page-align {
+    margin-left: 0px;
```

Comment:
nit: no need to specify units for 0s in CSS

Reply:
Yeah.